<ui version="4.0" stdsetdef="1" >
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>SSLCert</class>
  <widget class="QDialog" name="SSLCert" >
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>507</width>
        <height>369</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>SSL Cert</string>
    </property>
    <layout class="QVBoxLayout" >
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <layout class="QVBoxLayout" >
              <property name="margin" >
                <number>0</number>
              </property>
              <item>
                <widget class="QLabel" name="textLabel4" >
                  <property name="text" >
                    <string>Certificate Validation:</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="lb_valid" >
                  <property name="text" >
                    <string/>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="textLabel2" >
                  <property name="text" >
                    <string>Valid From:</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="lb_notBefore" >
                  <property name="text" >
                    <string/>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="textLabel3" >
                  <property name="text" >
                    <string>Valid Until:</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="lb_notAfter" >
                  <property name="text" >
                    <string/>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="textLabel1" >
                  <property name="text" >
                    <string>Serial Number:</string>
                  </property>
                </widget>
              </item>
              <item>
                <widget class="QLabel" name="lb_sn" >
                  <property name="text" >
                    <string/>
                  </property>
                </widget>
              </item>
              <item>
                <spacer name="spacer4" >
                  <property name="sizeHint" >
                    <size>
                      <width>20</width>
                      <height>106</height>
                    </size>
                  </property>
                  <property name="sizeType" >
                    <enum>Expanding</enum>
                  </property>
                  <property name="orientation" >
                    <enum>Vertical</enum>
                  </property>
                </spacer>
              </item>
            </layout>
          </item>
          <item>
            <widget class="Q3TextBrowser" name="tb_cert" >
              <property name="minimumSize" >
                <size>
                  <width>350</width>
                  <height>300</height>
                </size>
              </property>
              <property name="hScrollBarMode" >
                <enum>AlwaysOff</enum>
              </property>
              <property name="text" >
                <string/>
              </property>
              <property name="wrapPolicy" >
                <enum>AtWordOrDocumentBoundary</enum>
              </property>
            </widget>
          </item>
        </layout>
      </item>
      <item>
        <widget class="Line" name="line1" >
          <property name="frameShape" >
            <enum>QFrame::HLine</enum>
          </property>
          <property name="frameShadow" >
            <enum>QFrame::Sunken</enum>
          </property>
        </widget>
      </item>
      <item>
        <layout class="QHBoxLayout" >
          <property name="margin" >
            <number>0</number>
          </property>
          <item>
            <spacer name="spacer5" >
              <property name="sizeHint" >
                <size>
                  <width>421</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType" >
                <enum>Expanding</enum>
              </property>
              <property name="orientation" >
                <enum>Horizontal</enum>
              </property>
            </spacer>
          </item>
          <item>
            <widget class="IconButton" name="pb_close" >
              <property name="text" >
                <string>&amp;Close</string>
              </property>
            </widget>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
  <tabstops>
    <tabstop>tb_cert</tabstop>
  </tabstops>
</ui>
